Makefile.toml

```diff
@@ -8,7 +8,7 @@ BUILDSYS_ROOT_DIR = "${CARGO_MAKE_WORKING_DIRECTORY}"
 # For binary installation, this should be a released version (prefixed with a v,
 # for example v0.1.0). For the git sourcecode installation method, this can be
 # any git rev, e.g. a tag, sha, or branch name.
-TWOLITER_VERSION = "v0.4.1"
+TWOLITER_VERSION = "v0.4.2"
```

Comment:
This is not the correct version, we should be bumping to 0.4.3.

Reply:
woops, will fix